js/tic_tac_toe.js

```diff
@@ -1,2 +1,113 @@
-// Your code here
-// Use the jQuery
+$(document).ready(function(){
+  var $box_cell = $('.box_cell');
```

Comment:
You could probably set this in an init function in your game, but this is fine too :)